resources/compass/charts/metris/templates/deployment.yaml

```diff
@@ -24,6 +24,8 @@ spec:
     spec:
 {{- include "metris.imagePullSecrets" . | indent 6 }}
       serviceAccountName: {{ template "metris.fullname" . }}
+      nodeSelector: 
+        {{- toYaml .Values.deployment.nodeSelector | nindent 8 }}
```

Comment:
It should be more `{{- toYaml .Values.nodeSelector | nindent 8 }}`, there is no `deployment` in the value file.

Reply:
yes, sorry.. pasted too quick....